Dockerfile

```diff
@@ -34,30 +40,67 @@ RUN apt-get -y update && \
         libxpm-dev \
         make \
         openssl \
+        python3 \
+        python3-pip \
         python-dev \
         python-pip \
         xlibmesa-glu-dev \
         zlib1g-dev && \
-    rm -rf /var/lib/apt/lists/* && \
-    apt-get -y autoremove && \
-    apt-get -y clean
-
-ENV ROOTSYS /usr/local
-RUN git clone --quiet --depth 1 --branch v6-18-04 http://root.cern.ch/git/root.git /code/root && \
-    cd /code && \
-    mkdir _build && \
-    cd _build && \
-    cmake -Dcxx17=On -Dbuiltin_xrootd=On ../root && \
-    cmake --build . -- -j3 && \
-    cmake -P cmake_install.cmake && \
-    cd / && \
-    rm -rf /code
+    yum clean all && \
+    rm -rf /var/cache/yum
+
+RUN pip install --upgrade pip
+
+# gfal2-python bindings
+RUN curl -o /etc/yum.repos.d/gfal2-repo.repo https://dmc-repo.web.cern.ch/dmc-repo/dmc-el7.repo && \
+    git clone --branch v1.12.0 https://github.com/cern-fts/gfal2-python.git && \
+    cd gfal2-python/ && \
+    ./ci/fedora-packages.sh && \
+    cd packaging/ && \
+    RPMBUILD_SRC_EXTRA_FLAGS="--without docs --without python2" make srpm && \
+    yum-builddep -y python3-gfal2 && \
+    pip install gfal2-python
+
+RUN pip3 install rucio-clients==1.29.1
```

Comment:
It would be better to separate (i) move of the base image and (ii) addition of Rucio clients in two separate commits.  

The second is really optional to have in a ROOT environment image, and may not be wanted by everybody.  Hence calling for separate commits for separate feature changes here.